tests/affine.rs

```diff
@@ -0,0 +1,115 @@
+#[cfg(feature = "nalgebra_affine")]
+extern crate nalgebra;
+#[cfg(feature = "nalgebra_affine")]
+extern crate nifti;
+
+#[cfg(feature = "nalgebra_affine")]
+mod nalgebra_affine {
+    use nalgebra::Vector4;
+    use nifti::{affine::Affine4, NiftiHeader};
+
+    #[test]
+    fn affine() {
+        let mut header = NiftiHeader::default();
+        let affine = Affine4::from_diagonal(&Vector4::new(2.0, 2.0, 2.0, 1.0));
+        header.set_affine(&affine);
+        assert_eq!(affine, header.affine());
+        assert_eq!(header.sform_code, 2);
+        assert_eq!(header.qform_code, 0);
+    }
+
+    #[test]
+    fn sform() {
+        let mut header = NiftiHeader::default();
+        header.sform_code = 1;
+        header.qform_code = 0;
+        header.srow_x = [2.4, -0.0008, -0.0411765, -114.766396];
+        header.srow_y = [0.1, 2.4995277, 0.0485984, -97.420204];
+        header.srow_z = [0.4, -0.0485, 2.4991884, -89.12282];
+
+        let real_affine = Affine4::new(
+            2.4, -0.0008,    -0.0411765, -114.766396,
+            0.1,  2.4995277,  0.0485984,  -97.420204,
+            0.4, -0.0485,     2.4991884,  -89.12282,
+            0.0,  0.0,        0.0,          1.0
+        );
+        assert_eq!(header.affine(), real_affine);
```

Comment:
When performing equality checks between values with floating point numbers, it would be best to perform an approximate equality check (as in, assuming a small value margin, like in most other tests we have right now). Is there something we can use in `nalgebra` for this?

Reply:
Yes, they use `approx` in the `partial_cmp` free-[function](https://www.nalgebra.org/rustdoc/src/nalgebra/lib.rs.html#326-328). Will do.